packages/admin-ui-components/src/tooltip2/tooltip.tsx

```diff
@@ -9,19 +9,32 @@ import styles from "./tooltip.module.scss";
 export interface TooltipProps {
   children: React.ReactNode;
   theme?: "default" | "blue";
+  tableTitle?: boolean;
 }
 
 const cx = classNames.bind(styles);
 
 export const Tooltip = (props: TooltipProps & AntTooltipProps) => {
-  const { children, theme, overlayClassName } = props;
+  const { children, theme, overlayClassName, tableTitle } = props;
   const classes = cx(
     "tooltip-overlay",
     `crl-tooltip--theme-${theme}`,
     overlayClassName,
   );
+
+  const title = tableTitle ? (
```

Comment:
I don't see this in the source of `pkg/ui`. Why did we need to add it?

Reply:
we have different style for table title tooltip. to achieve that earlier we added <div/> with corresponding class(https://github.com/cockroachdb/cockroach/blob/master/pkg/ui/src/views/sessions/sessionsTableContent.tsx), but to remove duplications I moved this on tooltip component level.